xds/src/main/java/com/linecorp/centraldogma/xds/k8s/v1/XdsKubernetesService.java

```diff
@@ -204,12 +203,11 @@ private void validateKubernetesEndpointAndPush(
      * {@link KubernetesEndpointGroupBuilder#build()} blocks the execution thread.
      */
     public static CompletableFuture<KubernetesEndpointGroup> createKubernetesEndpointGroup(
-            ServiceEndpointWatcher watcher, MetaRepository metaRepository, String group,
-            String fileName, Executor executor) {
+            ServiceEndpointWatcher watcher, MetaRepository metaRepository, String group, String fileName) {
```

Comment:
If this method no longer blocks the execution thread, it may be worth updating the javadocs comment

Reply:
Thanks updated. 👍